app/firebase/functions/src/platforms/twitter/twitter.utils.ts

```diff
@@ -1,17 +1,18 @@
-import {
-  AppTweet,
-  AppTweetBase,
-  OPTIONAL_TWEET_FIELDS,
-  REQUIRED_TWEET_FIELDS,
-  TwitterThread,
-} from 'src/@shared/types/types.twitter';
 import {
   ApiResponseError,
   ApiV2Includes,
   TweetV2,
   UserV2,
 } from 'twitter-api-v2';
 
+import {
+  AppTweet,
+  AppTweetBase,
+  OPTIONAL_TWEET_FIELDS,
+  REQUIRED_TWEET_FIELDS,
+  TwitterThread,
+} from '../../@shared/types/types.twitter';
```

Comment:
@pepoospina this is where I was getting a strange error. If I kept the old path, I would get the error when running tests:
```
✖ ERROR: Error: Cannot find module 'src/@shared/types/types.twitter'
```

Which only came up if I was importing an `enum` (or a `const`) but worked fine for types and interfaces.

Reply:
It's a dumb error. The emulator forces us use relative paths, not absolute `/src`. You can replace `/src` with `../../ etc`